images/installer/Dockerfile

```diff
@@ -10,13 +10,13 @@ COPY images/installer/origin-extra-root /
 # install ansible and deps
 RUN INSTALL_PKGS="python-lxml python-dns pyOpenSSL python2-cryptography openssl java-1.8.0-openjdk-headless python2-passlib httpd-tools openssh-clients origin-clients" \
  && yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS \
- && EPEL_PKGS="python2-boto python2-boto3 python2-crypto google-cloud-sdk-183.0.0 which python2-pip.noarch" \
+ && EPEL_PKGS="python2-boto python2-boto3 python2-crypto google-cloud-sdk-183.0.0 which python2-pip.noarch python-scandir python2-packaging azure-cli" \
```

Comment:
@pschiffe @pweil-  
Per @sdodson request, we need to ensure that all packages added here are available in RHEL.

Reply:
Large enough for a card or nah? Let's make sure to not lose track.  /cc @jim-minter

Reply:
This is prerequisite before anything makes it into in OCP, we can patch the Dockerfile for OCP builds until they're available though.

Reply:
@sdodson if the dockerfile is already doing `pip install apache-libcloud`, any reason it couldn't do a `pip install azure-cli` too?
@kwoodson

Reply:
We're only able to do that there because we don't support it in OCP yet, if your requirements are the same then you're welcome to use pip.

Reply:
`python-scandir` and `python2-packaging` packages are from epel (`python-scandir` needs `gcc` if installed through epel), but `azure-cli` is from repo provided by Microsoft - https://github.com/openshift/openshift-ansible/pull/7690/files#diff-92e724033561f9568e169987b12f92b6